com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -566,6 +566,39 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
                 ConfigureCameraTarget(activeColorRenderTargetId, activeDepthRenderTargetId);
             }
 
+            bool renderingToBackBufferTarget = camera.targetTexture == null;
+
+            // If Camera Renders to backbuffer and MSAA Sample Count is larger than 1 we may have to disable/enable MSAA backbuffer
+            // The condition depends on whether the intermediate MSAA render targets has been created or not
+            if (cameraTargetDescriptor.msaaSamples > 1 && renderingToBackBufferTarget)
+            {
+                int qualitySettingsMsaaSampleCount = QualitySettings.antiAliasing > 0 ? QualitySettings.antiAliasing : 1;
+
+                if (createColorTexture)
+                {
+                    if (qualitySettingsMsaaSampleCount > 1)
+                    {
+                        QualitySettings.antiAliasing = 1;
```

Comment:
Code looks fine to me, however changing QualitySettings.antiAliasing causes changes in user project and that's not ok to change serialised data without user interaction. This will cause a regression related to this bug https://fogbugz.unity3d.com/f/cases/1195272/
 
Is it possible to add API to control backbuffer format instead of overriding quality settings?